app/styles/_mobirise.scss

```diff
@@ -186,8 +186,7 @@ $hamburger-color: $gray-darker;
 .mbr-navbar-column {
   display       : table-cell;
   vertical-align: middle;
-}
-
+}become-a-contributor
```

Comment:
1. there should be a new line after the curly brace and an empty line after that (for style)
2. you are missing a dot for a CSS class or a #hash mark for an ID